pandas/tests/tools/test_to_datetime.py

```diff
@@ -2895,10 +3020,7 @@ def test_incorrect_value_exception(self):
     )
     def test_to_datetime_out_of_bounds_with_format_arg(self, format, warning):
         # see gh-23830
-        msg = (
-            "Out of bounds nanosecond timestamp: 2417-10-27 00:00:00 "
-            "present at position 0"
-        )
+        msg = "Out of bounds nanosecond timestamp: 2417-10-27 00:00:00"
```

Comment:
Did we mean to lose the position part of this message?

Reply:
yup - I've opened https://github.com/pandas-dev/pandas/issues/50361 for this

I think it'd be easier to fixup the error message once the paths are unified, but I can address that one first if you prefer

Reply:
probably better to do that one first, actually, thanks for bringing this up - if the message is too long, then `re.search` will fail, but if it's too short, then it'll still pass (unless there is `'$'` at the end of the error message, which there typically isn't)

Reply:
for now I've just cherry-picked the commit from https://github.com/pandas-dev/pandas/pull/50366